R/derive_param_computed.R

```diff
@@ -117,21 +121,35 @@
 #' @param set_values_to Variables to be set
 #'
 #'   The specified variables are set to the specified values for the new
-#'   observations. For example `exprs(PARAMCD = "MAP")` defines the parameter
-#'   code for the new parameter.
+#'   observations. The values of variables of the parameters specified by
+#'   `parameters` can be accessed using `<variable name>.<parameter code>`,
+#'   e.g., `AVAL.SYSBP`. For example
```

Comment:
```suggestion
#'   . For example
```

Since the example is right after this can we just drop the `AVAL.SYSBP`?

Reply:
Updated